Packs/Axonius/Integrations/Axonius/TestData/Expected_data.py

```diff
@@ -0,0 +1,115 @@
+EXPECTED_USERS_SQS = [
```

Comment:
Run the following to add the secret to ignore file.

```bash
echo "dummy@axonius.com" > Packs/Axonius/.secrets-ignore
```

Conversely, remove the email completely.

Reply:
done